lldb/source/Plugins/ExpressionParser/Swift/SwiftExpressionParser.cpp

```diff
@@ -569,50 +573,54 @@ static void AddRequiredAliases(Block *block, lldb::StackFrameSP &stack_frame_sp,
 
   if (!self_type.IsValid() ||
       !self_type.GetTypeSystem()->SupportsLanguage(lldb::eLanguageTypeSwift))
-    return;
+    return Status("Could not add required aliases: self is not valid.");
 
   // Import before getting the unbound version, because the unbound
   // version may not be in the mangled name map.
 
   CompilerType imported_self_type = ImportType(swift_ast_context, self_type);
 
   if (!imported_self_type.IsValid())
-    return;
+    return Status(
+        "Could not add required aliases: imported self type is not valid.");
 
   auto *stack_frame = stack_frame_sp.get();
   if (bind_generic_types != lldb::eDontBind) {
     imported_self_type = swift_runtime->BindGenericTypeParameters(
         *stack_frame, imported_self_type);
     if (!imported_self_type)
-      return;
+      return Status("Could not add required aliases: could not bind self "
+                    "type's type parameters.");
   }
 
   {
     auto swift_type_system =
         imported_self_type.GetTypeSystem().dyn_cast_or_null<TypeSystemSwift>();
     if (!swift_type_system)
-      return;
+      return Status("Could not add required aliases: unexpected type system.");
```

Comment:
I took a guess what thing is expecting a type.

```suggestion
      return Status("Unable to add the aliases the expression needs because of a system type the Swift expression parser doesn't expect.");
```

If the word "Unable" isn't a valid option, then I suggest the contraction "Couldn't".

Reply:
Type system here is an LLDB concept, which indicates what type of language this type belongs to (for example, the Swift type system, the C++ type system, etc).
 
Maybe: "Unable to add the aliases the expression needs because self is not a Swift type."?

Reply:
I like it and it's more concise than my draft.